plugins/redfish/fu-redfish-device.c

```diff
@@ -434,7 +474,7 @@ fu_redfish_device_probe(FuDevice *dev, GError **error)
 		if (tmp != NULL)
 			fu_device_set_backend_id(dev, tmp);
 	}
-	fu_device_add_instance_str(dev, "ID", fu_device_get_backend_id(dev));
```

Comment:
That one. It is not necessary on Dell servers (and is even counter productive, because IDs contains the versions) and may or may not be necessary on HPE servers (due to the issue I opened, I still need to test)

Reply:
Just tested on HPE, this does not seem needed

Reply:
Yup, on Lenovo they're just using SOFTWAREID -- I think we can remove ID